### PR TITLE
Added capability to login in an iframe

### PR DESCRIPTION
**Pull request by vishalshrm539**
If iframeLoginUI property within sdk-config.json is set to true then we will be able to login using an iframe instead of a popup window for all the re-authentications i.e. all authentications except for the first one.

### CODE REVIEW

**Review by JEAxero — COMMENTED** [review]
I'll defer to Vinod for his auth expertise but these changes look ok to me.

File: src/helpers/authManager.js
Comment (vrseraphin) [inline src/helpers/authManager.js:124]: Suggest looking for this optional setting and coping with it being a boolean of true or the string "true"

File: sdk-config.json
Comment (vrseraphin) [inline sdk-config.json:7]: Suggest adding iframeLoginUI as an optional attribute (and not have it explicitly listed).  Rather just have info for it in README

File: sdk-config.json
Comment (vrseraphin) [inline sdk-config.json:9]: Suggest not having it in config file by default and document that someone could add it...but only if they are working with an identity provider login screen that will both display well in an iFrame as well as allow submitting in the iframe.

File: sdk-config.json
Comment (JEAxero) [reply sdk-config.json:7]: I don't think we have complete documentation for the sdk-config.json file options in either the SDK Guide or the README. But the SDK Guide info is more complete than what's in the README. And, since the explanation for all of the various options will get pretty lengthy, I suggest that we have the README point to a (new) complete page of sdk-config.json options that we'll put in the **docs** folder. I'll create a story for that doc work but I don't think we need to hold up merging this PR to wait for its completion (once the other comments on this PR have been addressed).

**Comment by vishalshrm539** [comment]
As per Vinod's suggestion, made 'iframeLoginUI' an optional parameter in sdk-config.js file, if specified then only will be utilised.

**Review by vrseraphin — APPROVED** [review]

**Pull request merged**
